rcsoccersim.sh

```diff
@@ -32,11 +32,13 @@ if [ "$#" -eq 0 ]; then
   exit 1
 fi
 
-for arg in "$@"; do
+ARGS=${1}
+IFS=' ' #set the internal field separator to 
+read -ra TOKENS <<< "$ARGS"
```

Comment:
Qual erro tava ocorrendo?

Reply:
Eu precisava passar os paramêtros por meio de uma string na hora da instalação tipo "rcssserver rcssmonitor", o for só instalava o primeiro pacote e não iterava entre as duas palavras. Daí eu tive que transformar essa string em um vetor de strings e rolou

Reply:
Pode mandar o comportamento antes? Iterar da forma anterior tava sendo utlizada [aqui](https://github.com/robocin/containers/blob/66862dbb0d158b4de8a351f05d2a6d6e749725d1/devcontainers/scripts/robocin-install.sh#L33) sem problemas

Reply:
Vei tentei replicar aqui mas não consegui, ta funcionando do outro jeito mesmo. Vou voltar o arquivo pra forma anterior